sh/tests/set-env-vars_test.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+submitted=$(bash student/set-env-vars.sh)
```

Comment:
it's bad idea to compare the all env vars, maybe cause you can't control the output of all of them
i propose to grep by MY_ when you print and ask the student to do that in his script

Reply:
Done